docs/src/components/header/MainLayoutHeader.vue

```diff
@@ -481,12 +487,24 @@ export default defineComponent({
     // on these particular viewports, we reduce the padding on the dense header to prevent
     // icon or some items from collapsing.
     const denseHeaderNavItemPadding = computed(() => {
-      if ($q.screen.width >= $q.screen.sizes.lg && $q.screen.width < 1500) {
+      if ($q.screen.width >= $q.screen.sizes.md && $q.screen.width < 1500) {
+        return 'xs sm'
+      }
+
+      if ($q.screen.width >= 769 && $q.screen.width < 820) {
+        return 'xs sm'
+      }
+
+      if ($q.screen.width >= $q.screen.sizes.sm && $q.screen.width < 685) {
         return 'xs sm'
       }
 
       return 'xs md'
     })
+    // hide separator on this specific viewport to prevent version dropdown from collapsing
+    const showDenseSeparator = computed(() => {
```

Comment:
Have you discussed these changes with Nick? The effect when removing the separator doesn't really look nice